### PR TITLE
[KHP3-6545 ] Implement ability to move patient from radiology to new queue

### PR DESCRIPTION
**Pull request by its-kios09**
## Requirements
- [ ] This PR has a title that briefly describes the work done including a [conventional commit](https://o3-dev.docs.openmrs.org/#/getting_started/contributing?id=your-pr-title-should-indicate-the-type-of-change-it-is) type prefix and a Jira ticket number if applicable. See existing PR titles for inspiration.

#### For changes to apps
- [ ]  My work conforms to the [**OpenMRS 3.0 Styleguide**](https://om.rs/styleguide) and [**design documentation**](https://zeroheight.com/23a080e38/p/880723-introduction).

#### If applicable
- [ ] My work includes tests or is validated by existing tests.

## Screenshots

radiologyie

## Related Issue

see: https://thepalladiumgroup.atlassian.net/browse/KHP3-6545

### CODE REVIEW

File: src/radiology-tabs/common/groupedOrdersTable.component.tsx [hunk 1]
```diff
@@ -68,13 +69,34 @@ const GroupedOrdersTable: React.FC<GroupedOrdersTableProps> = (props) => {
...
-    { id: 0, header: t("patient", "Patient"), key: "patientname" },
-    { id: 1, header: t("totalorders", "Total Orders"), key: "totalorders" },
-  ];
+  const tableColumns = useMemo(() => {
+    const showActionColumn = workListEntries.some(
+      (order) => order.fulfillerStatus === "COMPLETED"
+    );
+
+    const columns = [
+      { id: 0, header: t("patient", "Patient"), key: "patientname" },
```
Comment (makombe) [inline src/radiology-tabs/common/groupedOrdersTable.component.tsx:86]:
```suggestion
      { id: 0, header: t("patient", "Patient Name"), key: "patientname" },
```

File: src/radiology-tabs/common/groupedOrdersTable.component.tsx [hunk 2]
```diff
@@ -68,13 +69,34 @@ const GroupedOrdersTable: React.FC<GroupedOrdersTableProps> = (props) => {
...
+      { id: 1, header: t("totalorders", "Total Orders"), key: "totalorders" },
+    ];
+
+    if (showActionColumn) {
+      columns.push({
+        id: 2,
+        header: t("actionButton", "Action"),
```
Comment (makombe) [inline src/radiology-tabs/common/groupedOrdersTable.component.tsx:93]:
```suggestion
        header: t("action", "Action"),
```

File: src/radiology-tabs/test-ordered/transition-patient-new-queue/transition-latest-queue-entry-button.scss [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+.addPatientToQueue {
+  --cds-layout-size-height-context: var(--cds-layout-size-height-sm, 2rem);
+  --cds-layout-size-height: var(--cds-layout-size-height-context);
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  text-align: center;
+  padding: 0 layout.$spacing-04;
+  gap: 1rem;
```
Comment (makombe) [inline src/radiology-tabs/test-ordered/transition-patient-new-queue/transition-latest-queue-entry-button.scss:12]:
```suggestion
  gap: layout.$spacing-05;
```

**Review by makombe — COMMENTED** [review]
Thanks @its-kios09. I have left some few comments.

File: src/radiology-tabs/common/groupedOrdersTable.component.tsx (hunk 2 above)
Comment (its-kios09) [reply src/radiology-tabs/common/groupedOrdersTable.component.tsx]: same

**Review by makombe — APPROVED** [review]

**Pull request merged**